web/cobrands/buckinghamshire/layout.scss

```diff
@@ -164,3 +164,15 @@ body, .content {
   font-size: 1em;
   padding: 20px;
 }
+
+// Adjust the size dropdown button and menu for mappage
+.mappage {
+  .multi-select-button {
+    max-width: 80px;
+  }
+  
+  .multi-select-menu {
+    left: -55px;
+    width: 240px !important;
```

Comment:
I think this line is unnecessary - with the left of the line above, the code automatically calculates a greater width for the box. 240px is too wide with the live category data, as the scrollbar underlaps the edge of the map:
![image](https://user-images.githubusercontent.com/154364/157723843-45f6c711-7af0-4350-9af7-2a9c051adef9.png)
But without this line it works it out as c. 215px and looks okay.

Reply:
You are right, it actually look much better. I just pushed a commit deleting that line. Thanks =)